dlt/common/normalizers/json/relational.py

```diff
@@ -422,12 +427,39 @@ def _normalize_prop(
             validator_f=column_name_validator(schema.naming),
         )
 
+    #
+    # Cached helper methods for all operations that are called often
+    #
+    @staticmethod
+    @lru_cache(maxsize=None)
```

Comment:
this adds a considerable speed boost. we could also consider adding caching support on the naming and not here so that all normalizers and other places can benefit, I'm not quite sure if there are other places where this gets called as often as here though.

Reply:
this is interesting because this functions is already cached. are you using `snake_case`? convention? please look at the underlying code again